ios/FluentUI/Pill Button Bar/PillButtonBarTokens.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFPillButtonBarTokens: MSFTokensBase {
+    public var cornerRadius: CGFloat!
+    public var maxButtonsSpacing: CGFloat!
+    public var minButtonsSpacing: CGFloat!
+    public var minButtonVisibleWidth: CGFloat!
+    public var minButtonWidth: CGFloat!
+    public var minHeight: CGFloat!
+    public var sideInset: CGFloat!
+    public var hoverBackgroundColor: UIColor!
+    public var onBrandHoverBackgroundColor: UIColor!
+
+    override init() {
+        super.init()
+
+        self.themeAware = true
+
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
+    }
+
+    override func updateForCurrentTheme() {
+        let currentTheme = theme
+        let appearanceProxy = currentTheme.MSFPillButtonBarTokens
```

Comment:
You probably don't need currentTheme if you're only using it to set up appearanceProxy.